build.gradle

```diff
@@ -13,7 +13,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.3.0'
+        classpath 'com.android.tools.build:gradle:2.3.1'
```

Comment:
@MarcinMoskala this change is not related to this pull request so reverting and adding this change as another PR (maybe there's more dependencies to update like Kotlin plugin) would be a great option.